src/state.rs

```diff
@@ -52,6 +52,18 @@ impl<T: 'static> Flag<T> {
     /// `is_present()` will be false and `.flag` will refer to the default
     /// value.
     pub fn is_present(&self) -> bool {
+        self.present.load(Ordering::SeqCst) != 0
+    }
+
+    /// Count number of times an option is repeated on the command line.
+    ///
+    /// Useful to display verbosity or debug level by repeating a boolean flag
+    /// several times. For example, `-vv` for "very verbose" (repeat count 2)
+    /// or `-ddd` for debug level 3.
+    ///
+    /// The count wraps around to zero if an option is repeated on the
+    /// command line 256 times.
```

Comment:
Would it be easy to print a meaningful error messages instead?

Reply:
Alternatively, one can upgrade the counter  from `AtomicU8` to `AtomicU32`. This will guarantee that overflow never happens due to the constraints on the command line length. In x64 Linux (bash) `getconf ARG_MAX` reports back 2MB; on MacOS (bash and zsh)  it is 256KB.

Reply:
Pushed a new commit to the same PR branch that upgraded the repeat counter from `AtomicU8` to `AtomicU32`. Its overflow will never happen due to the command line length limitations.